plyer/platforms/macosx/audio.py

```diff
@@ -0,0 +1,81 @@
+from plyer.facades import Audio
+from pyobjus.dylib_manager import load_framework, INCLUDE
+from pyobjus import autoclass
+from os.path import join, expanduser
+
+load_framework(INCLUDE.Foundation)
+load_framework(INCLUDE.AVFoundation)
+
+AVAudioPlayer = autoclass("AVAudioPlayer")
+AVAudioRecorder = autoclass("AVAudioRecorder")
+AVAudioFormat = autoclass("AVAudioFormat")
+NSString = autoclass('NSString')
+NSURL = autoclass('NSURL')
+
+class OSXAudio(Audio):
+    def __init__(self, file_path=None):
+        default_path = join(expanduser('~'), 'Desktop', 'audio.wav')
```

Comment:
You should not pass a default value that point on nothing :) Should be None by default.

Reply:
Tried without passing a value of `None`, but I started getting the following error:
`>>> from plyer import audio
>>> audio.start()
Traceback (most recent call last):
  File "/Users/maroskovac/Dev/plyer/plyer/utils.py", line 96, in _ensure_obj
    obj = mod.instance()
  File "/Users/maroskovac/Dev/plyer/plyer/platforms/macosx/audio.py", line 82, in instance
    return OSXAudio()
TypeError: __init__() takes exactly 2 arguments (1 given)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/maroskovac/Dev/plyer/plyer/facades/audio.py", line 65, in start
    self._start()
  File "/Users/maroskovac/Dev/plyer/plyer/facades/audio.py", line 98, in _start
    raise NotImplementedError()
NotImplementedError`